pkg/controller/controller.go

```diff
@@ -406,8 +417,7 @@ func (ctrl *resizeController) resizePVC(pvc *v1.PersistentVolumeClaim, pv *v1.Pe
 		// Record an event to indicate that resize operation is failed.
 		ctrl.eventRecorder.Eventf(pvc, v1.EventTypeWarning, util.VolumeResizeFailed, err.Error())
 	}
-
-	return err
+	return nil
```

Comment:
Why is this changed to nil?

Reply:
not sure how this happeend. Added a unit test to ensure this does not happen. good catch.